md/profileCreation.md

```diff
@@ -0,0 +1,79 @@
+# Profile Editor
+
+## Definition
+
+Basically, a **profile** is an XML file. It contains the profile name, the mapping with the organization (which users / groups / roles / memberships have this profile) and the portal menu entries (menus accessible to a user logged in the portal with this profile).  
+A profile must be imported onto the portal, in the **Profiles** tab, to enable the profile (you need to connect with the **Administrator** profile to do it).  
+Before Bonita 7.6, this portal tab was the only way to create and manage profiles.  
```

Comment:
Since Bonita 7.6.0, Bonita Studio offers this feature, for better packaging and deployment.